packages/mwp-app-route-plugin/src/route.js

```diff
@@ -12,7 +15,13 @@ export const onPreResponse = {
 	 */
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
-
+		if (request.query) {
```

Comment:
I _think_ you can always count on `request.query` existing, even if it's an empty object.